README.md

```diff
@@ -114,6 +114,11 @@ Rasa uses Poetry for packaging and dependency management. If you want to build i
 you have to install Poetry first. Please follow
 [the official guide](https://python-poetry.org/docs/#installation) to see all possible options.
 
+To update an existing poetry version to the version 1.4.2, currently used in rasa, run:
+```shell
+    poetry self update 1.4.2
```

Comment:
Tiny suggestion: instead of hardcoding the version, maybe we can link to the the variable stored in the `txt` file under github workflows?

Reply:
Done

Reply:
Nice one 🎉